hugo/assets/ts/widgets/search-filter.ts

```diff
@@ -83,20 +83,35 @@ export class SearchFilter extends BaseWidget {
             return `<li class="filter__item">
                 <button class="filter__link${ isSelected ? ' is-selected' : '' }"
                 type="button" data-value="${ item.name }">
-                    ${ item.color ? `<span class="filter__color filter__color--${ item.color }"></span>` : ''}
+                    ${ item.color ? `<span class="filter__color filter__color--${ item.color }"></span>` : '' }
                     ${ item.name }
                 </button>
             </li>`;
         });
 
+        if (this.selectedItems.length > 0) {
+            html.push(
+                `<li class="filter__item">
+                    <button class="filter__link filter__link--clear" type="button" data-clear=true>
+                        <span>Clear all items</span>
```

Comment:
Maybe "Reset items" or "Reset filters" is more clear?
Also I'm not sure about that it's center aligned since everything else is aligned to the left: maybe check with Peter what he thinks

Reply:
This is what Peter and I came up with yesterday